app/assets/javascripts/discourse/app/instance-initializers/banner.js

```diff
@@ -26,5 +28,18 @@ export default {
     } else {
       this.site.set("banner", null);
     }
+  }
+}
+
+export default {
+  after: "message-bus",
+
+  initialize(owner) {
+    this.instance = new BannerInit(owner);
+  },
+
+  teardown() {
+    this.instance.teardown();
+    this.instance = null;
```

Comment:
Can you explain to me why this is necessary? Is it a way for this to be collected faster?

Reply:
If we don't set it to `null` here, then it will persist until `initialize()` is called in the next test. Not the end of the world, but feels a little nicer to leave a clean state at the end of each test run